test/native/src/test/resources/seata-script-client-conf-file.conf

```diff
@@ -20,3 +20,9 @@
 service {
     vgroupMapping.default_tx_group = "default"
 }
+# TODO This is not a reasonable configuration, just affected by https://github.com/apache/incubator-seata/pull/6661. Pending investigation.
+client {
+    rm {
+        tableMetaCheckEnable = "false"
+    }
+}
```

Comment:
- I have opened https://github.com/apache/incubator-seata/issues/7042 . I don't believe this is sensible default behavior for Seata Client.